README.md

```diff
@@ -16,36 +19,37 @@ _[Demo and API Docs](https://elements.polymer-project.org/elements/iron-demo-hel
 
 ##&lt;demo-snippet&gt;
 
-
 `demo-snippet` is a helper element that displays the source of a code snippet and
 its rendered demo. It can be used for both native elements and
 Polymer elements.
 
-    Example of a native element demo
+```html
```

Comment:
Yes! I know that it's legally, but I think that this text isn't part of the code samples.

Okey! :+1: